src/Umbraco.Web/Editors/PreviewController.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Web.Mvc;
+using Umbraco.Web.Features;
+using Umbraco.Web.Mvc;
+
+namespace Umbraco.Web.Editors
+{
+    [UmbracoAuthorize]
+    [DisableBrowserCache]
+    public class PreviewController : Controller
```

Comment:
@clausjensen Why isn't this authorized? this route will be public. I understand that preview used to work with an iframe that was authorized and perhaps that is still the case, but we should probably still not render this route as unauthorized.

Reply:
ah, never mind, see didn't see the attribute. perfect.